near-sdk/src/store/unordered_map/mod.rs

```diff
@@ -630,6 +630,23 @@ where
     }
 }
 
+impl<K, V, H> UnorderedMap<K, V, H>
+where
+    K: BorshSerialize + BorshDeserialize + Ord + Clone,
+    V: BorshSerialize + BorshDeserialize,
+    H: ToKey,
+{
+    pub fn defrag(&mut self) {
```

Comment:
It definitely needs documentation with an example of where it is useful and why (best practices).

Reply:
I did my best, but would appreciate a look!

Reply:
The documentation looks great.

P.S. It would be great if we also extend the documentation on `.remove` and `.remove_entry` with the reference to `.defrag`

Reply:
Done!